greenkeeper.json

```diff
@@ -0,0 +1,22 @@
+{
+  "groups": {
+    "default": {
+      "packages": [
```

Comment:
Rebase from master. This commit has been already merged.

Reply:
I rebased from webpack-cli master but can't figure out why this commit pops up.